examples/otlptrace/build.gradle

```diff
@@ -20,28 +20,31 @@ plugins {
 // examples are not published, so version can be hardcoded
 version = '0.1.0'
 
-mainClassName = 'com.google.cloud.opentelemetry.example.otlptrace.OTLPTraceExample'
-
 description = 'Example for OTLP exporter'
 
 dependencies {
 	implementation(libraries.opentelemetry_api)
 	implementation(libraries.opentelemetry_sdk)
 	implementation(libraries.opentelemetry_otlp_exporter)
 	implementation(libraries.opentelemetry_sdk_autoconf)
-	implementation(libraries.google_auth)
+	// the shaded variant name is empty
+	implementation("${libraries.opentelemetry_gcp_auth_extension}:")
```

Comment:
🙈 

Not sure if it's only .kts, but is there a way to pass named arguments so you could make the empty string more explicit?

Reply:
Yeah for some reason,
```groovy
implementation group: 'io.opentelemetry.contrib', name: 'opentelemetry-gcp-auth-extension', version: '1.43.0-alpha', classifier: ''
```
is not working. 

I made a change that makes the intentional use of empty classifier more explicit. This is temporary and will be fixed in the next release of the auth extension.